x/storage/keeper/keeper.go

```diff
@@ -1649,3 +1664,23 @@ func (k Keeper) garbageCollectionForResource(ctx sdk.Context, deleteStalePolicie
 	}
 	return deletedTotal, true
 }
+
+func (k Keeper) setQuotaUpdateTime(ctx sdk.Context, bucketId types.Uint, timestamp uint64) {
+	store := ctx.KVStore(k.storeKey)
+	bz := make([]byte, 8)
+	binary.BigEndian.PutUint64(bz, timestamp)
+	store.Set(types.GetQuotaKey(bucketId), bz)
```

Comment:
Can the quota update time be stored directly in bucketInfo? It didn't feel necessary to add a separate key

Reply:
Or, I think we should separate some frequently modified fields from bucketInfo as a separate key-value.

Reply:
the update frequency of `quote` is very low, or event no update at all. 
currently, if there is no update, it will not be stored (just use bucket' creation time).